homeassistant/components/fan/__init__.py

```diff
@@ -235,6 +235,7 @@ class FanEntity(ToggleEntity):
     """Base class for fan entities."""
 
     entity_description: FanEntityDescription
+    _attr_supported_features = 0
```

Comment:
The use of shorthand property for Fans is not supported.

See also: 

![image](https://user-images.githubusercontent.com/195327/141649729-f9e3c753-7106-4635-8c57-abf62ebe51ab.png)

https://developers.home-assistant.io/docs/core/entity/fan

I think it is great if we can implement it, however, we should not implement a single property in that case (which this PR does).

Reply:
as always, it make absolutely sense what you are saying 😄 
i will close this PR for now and come back with a new one about implementing shorthand properties in the fan platform

Reply:
It's confusing though since supported_features is implemented by the base entity class. So the property is there. And the implementation is there. _attr_name, works, _attr_device_info works, so why should we explicitly block supported features?

Reply:
@elupus Agree. However, the FanEntity should not be used in combination with shorthand attribute notations, as the implementation is incomplete. Hence the developer note.

Adding this PR, is not going to change that.